src/routes/auth.route.js

```diff
@@ -9,6 +9,9 @@ import { checkPassword } from '../helpers/owasp';
 const router = express.Router(); // eslint-disable-line new-cap
 
 router.route('/login')
+// Setting `AUTH_SERVICE_REQUIRE_ACCOUNT_VERIFICATION` or
```

Comment:
All of these comments about what these various controllers should do should not be here and should instead be in their respective controller files.